nav2_costmap_2d/src/costmap_2d_cloud.cpp

```diff
@@ -185,7 +185,7 @@ void voxelCallback(const nav2_msgs::msg::VoxelGrid::ConstSharedPtr grid)
       // uint32_t a = g_colors_a[c.status] * 255.0;
 
       uint32_t col = (r << 16) | (g << 8) | b;
-      cval = *reinterpret_cast<float *>(&col);
+      memcpy(&cval, &col, sizeof col);
```

Comment:
Same here. I doubt `cval` is valid, and it is definitely undesirable to do the extra memcpy.

Reply:
Ditto. I take my comment back. :-)